aspnet/security/introduction-to-aspnet-identity.rst

```diff
@@ -0,0 +1,100 @@
+
+The purpose of this article is to give you a high level overview of ASP.NET Identity; you can follow it step by step or just read the details. For more detailed instructions about creating apps using ASP.NET Identity, see the Next Steps section at the end of this article.
+
+1. Create an ASP.NET MVC application with Individual Accounts. You can use ASP.NET Identity in an ASP.NET MVC application, as well as other ASP.NET frameworks, such as Web Forms, Web API, SignalR, etc. In this article, you will start with an ASP.NET MVC application. In Visual Studio, select **File** -> **New** -> **Project**. Then, select the **ASP.NET Web Appication** from the **New Project** dialog box. Continue by selecting a **Web Appication** with **Individual User Accounts** as the authenication method.
+
+	.. image:: introduction-to-aspnet-identity/_static/01-mvc.png
+	
+2. The created project contains the following ASP.NET Identity package.
+
+- ``Microsoft.AspNet.Identity.EntityFramework``
+ The `Microsoft.AspNet.Identity.EntityFramework <http://www.nuget.org/packages/Microsoft.AspNet.Identity.EntityFramework/>`_ package has the Entity Framework implementation of ASP.NET Identity which will persist the ASP.NET Identity data and schema to SQL Server. 
+
+.. note:: In Visual Studio, you can view NuGet packages details by selecting **Tools** -> **NuGet Package Manager** -> **Manage NuGet Packages for Solution**.
+ 
+3. Creating a user.
+
+ Launch the application from Visual Studio (**Debug** -> **Start Debugging**) and then click on the **Register** link in the browser to create a user. The following image shows the *Register* page which collects the user name and password.
+
+	.. image:: introduction-to-aspnet-identity/_static/02-reg.png
+
+When the user clicks the **Register** button, the ``Register`` action of the ``AccountController`` creates the user by calling ``CreateAsync`` function of the ``UserManager`` object, as shown below:
+
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 101-127
+	:emphasize-lines: 4,10
+	:dedent: 8
+	
+4. Log in.
+ If the user was successfully created, they are logged in by the ``SignInAsync`` method, also contained in the ``Register`` action.
+	
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 101-127
+	:emphasize-lines: 19
+	:dedent: 8
+
+The above ``SignInAsync`` method calls the below ``SignInAsync`` task, which is contained in the ``SignInManager`` class. The ``SignInAsync`` method is overloaded. The ``authenticationProperties`` parameter that can be added to the call. This parameter is a dictionary used to store state values about the authentication session. These values are applied to the login and authentication cookie. 
+
```

Comment:
It would be nice at this point to show how the user identity is now available inside controller actions. e.g., if you set a breakpoint inside `HomeController.Index` and then navigate to the home page, you can see the claims by looking at `User.Claims`.